config-model/src/main/java/com/yahoo/vespa/model/builder/xml/dom/chains/search/DomProviderBuilder.java

```diff
@@ -62,10 +59,6 @@
             connectionPoolTimeout = readConnectionPoolTimeout(providerElement);
             retries = readRetries(providerElement);
             nodes = readNodes(providerElement);
-            certificateApplicationId = readCertificateApplicationId(providerElement);
```

Comment:
Can these `readCertificate*` methods be removed as well?

Reply:
Yes, done - thanks